node/.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 !typings/**/*.d.ts
 !test/**/*.js
 !tspublish.js
+README.md
```

Comment:
Specify it as `./README.md` to prevent excluding any other READMEs that might be added in the tree.